Diagnostic/diagnostic.py

```diff
@@ -361,27 +348,6 @@ def main(command):
                 hutil.do_status_report(g_ext_op_type, "error", '-1', "Install failed")
                 return
 
-            #Start the Telegraf and ME services on Enable after installation is complete
```

Comment:
No reason to be starting telegraf/ME yet during install, this is going to happen in enable